newspapers.com.js

```diff
@@ -0,0 +1,104 @@
+function detectWeb(doc, url) {  
+	return "newspaperArticle";
+}
+function doWeb(doc, url) { 
+	var namespace = doc.documentElement.namespaceURI;
+	var nsResolver = namespace ? function(prefix) {
+		if (prefix == 'x') return namespace; else return null;
+	} : null;
+
+	var newItem = new Zotero.Item("newspaperArticle");
+	var metaArr = new Object();
+	var metaTags = doc.getElementsByTagName("meta");
+	for (var i = 0 ; i < metaTags.length ; i++) {
+		metaArr[metaTags[i].getAttribute("property")] = metaTags[i].getAttribute("content");
+	}
+	/*
+	<meta property="og:title" content="Oct 11 1939 Detroit Free Press bork - on Newspapers.com">
+	*/
+	newItem.title = metaArr["og:title"].replace(/(.+?) -( on)? Newspapers.com/, "$1");
+	newItem.url = metaArr["og:url"];
+	
+	if (newItem.title.indexOf('/') >= 0) {
+		var tokens = newItem.title.split("/");
+		var author = tokens[1];
+		newItem.title = tokens[0].trim();
+	    if (author.match("; ")) {
+	      var authors = author.split("; ");
+	      for (var i in authors) {
+	        newItem.creators.push(Zotero.Utilities.cleanAuthor(authors[i], "author"));
+	      }
+	    } else {
+	      newItem.creators.push(Zotero.Utilities.cleanAuthor(author, "author"));
+	    }
+	}
+
+	/*
+	<span id="spotBody" class="disc-body">bork bork bork</span>
+	*/
+	newItem.abstractNote = doc.getElementById("spotBody").innerHTML;
+	
+	/*
+	<meta property="og:image" content="https://img0.newspapers.com/img/img?id=97710064&width=557&height=4616&crop=1150_215_589_4971&rotation=0&brightness=0&contrast=0&invert=0&ts=1467779959&h=e478152fd53dd7afc4e72a18c1dad4ea">
+	*/
+	imageurl = metaArr["og:image"];
+	newItem.attachments = [{
+		url: imageurl,
+		title: "Image",
+		mimeType: "image/jpeg"
+	}];
+	
+	/*
+	<span id="printlocation">
+		<a href="/browse/#hoC0Us-e7">The Evening News (Harrisburg, Pennsylvania)</a> &nbsp;
+		<a href="/browse/#hoC0Us-e77MhXcywhyDn5s2CcpUK9XEjt">28 Jun 1929, Fri</a> &nbsp;
+		<a href="/image/#63217003">Page 13</a>
+	</span>
+
+	or 
+
+	<span id="printlocation">
+<a href="/browse/#hO4iD0U5g">Reno Gazette-Journal (Reno, Nevada)</a> &nbsp;
+<a href="/browse/#hO4iD0U5gKAOus8IgtaE4K1d71AL1Hvf-">21 Oct 1939, Sat</a> &nbsp;
+<a href="/browse/#hO4iD0U5gKAOus8IgtaE4K1d71AL1Hvf-qe4sECZ-">Main Edition</a> &nbsp;
+<a href="/image/#148827895">Page 4</a>
+</span>
+	*/
```

Comment:
I guess you want to make a point that `citation` can contain 3 or 4 nodes and how they will then look like. Then you should write that in some sentence(s). And fix the indentation if the example should stay here.